cli/tests/integration/check_tests.rs

```diff
@@ -194,34 +194,34 @@ fn typecheck_declarations_unstable() {
   output.assert_exit_code(0);
 }
 
-#[test]
-fn typecheck_core() {
-  let context = TestContext::default();
-  let deno_dir = context.deno_dir();
-  let test_file = deno_dir.path().join("test_deno_core_types.ts");
-  std::fs::write(
-    &test_file,
-    format!(
-      "import \"{}\";",
-      deno_core::resolve_path(
-        &util::root_path()
-          .join("core")
-          .join("lib.deno_core.d.ts")
-          .to_string(),
-        &std::env::current_dir().unwrap()
-      )
-      .unwrap()
-    ),
-  )
-  .unwrap();
-
-  let args = vec!["run".to_string(), test_file.to_string()];
-  let output = context.new_command().args_vec(args).split_output().run();
-
-  println!("stdout: {}", output.stdout());
-  println!("stderr: {}", output.stderr());
-  output.assert_exit_code(0);
-}
+// #[test]
+// fn typecheck_core() {
+//   let context = TestContext::default();
+//   let deno_dir = context.deno_dir();
+//   let test_file = deno_dir.path().join("test_deno_core_types.ts");
+//   std::fs::write(
+//     &test_file,
+//     format!(
+//       "import \"{}\";",
+//       deno_core::resolve_path(
+//         &util::root_path()
+//           .join("core")
+//           .join("lib.deno_core.d.ts")
+//           .to_string(),
+//         &std::env::current_dir().unwrap()
+//       )
+//       .unwrap()
+//     ),
```

Comment:
Are you gonna remove this?

Reply:
I'll get rid of it but I think it needs to find a new home.